### ISSUE TITLE
Give logo image an `alt` attribute

### ISSUE THREAD

**Issue by neel-a-mehta** [seattle-uat/civiform#1473]
The Seattle logo image, found in the header bar, lacks an `alt` attribute, which is important for accessibility. Let's give it one.

![image](https://user-images.githubusercontent.com/77128518/123184362-614b7100-d448-11eb-91e8-ed2a5fd18d99.png)

## Original tester's report

> ACTUAL RESULT:
Voice Over: Screen reader is announcing 'Seattle' image logo as 'Seattle-logo_horizontal_blue-white_small.png, image'
ChromeVox: Screen reader is announcing 'Seattle' image logo as 'To get missimg image description, open the context menu. Image'
JAWS/Chrome: Screen reader is announcing 'Seattle' image logo as 'To get missimg image description, open the context menu. Unlabeled graphic'
> EXPECTED RESULT: If it is a decorative image, screen reader focus should not be set on the 'Seattle' logo image. If the image is conveying valid information, provide appropriate description for the 'Seattle' image.
